Header header

# Parameters of intensity sensor model
float64 reflectance_param
uint32 init_value
